smalltalksrc/VMMaker/StackInterpreter.class.st

```diff
@@ -5593,11 +5593,13 @@ StackInterpreter >> extStoreReceiverVariableBytecode [
 	| variableIndex |
 	<compilationInfo: #isInstVarRef>
 	<compilationInfo: #isMappedIfImmutability>
+	<needsFrameIfImmutability: 0>
+
 	variableIndex := self fetchByte + (extA << 8).
-	extA := 0.
 	self
 		storeMaybeContextReceiverVariable: variableIndex
 		withValue: self stackTop.
+	extA := 0.
```

Comment:
Hmm I see. Move side effects down. Does it really change compilation?

Reply:
Mmmmm I don't remember, maybe it has not effect and was just a test. I'll check 🤔